packages/design-system-docs/src/pages/components/FilterTag/_FilterTag.docs.scss

```diff
@@ -0,0 +1,45 @@
+/*
+Filter Tag
+
+Filter Tags are used to display dismissible filter chips
+
+Markup: filtertag.example.html
+
+Style guide: components.filtertag
+*/
+
+/*
+`<FilterTag>`
+
+@react-example FilterTag.example.jsx
+
+@react-props FilterTag.jsx
+
+Style guide: components.filtertag.react
+*/
+
+/*
+---
+
+### When to use
+
+- To filter results with one or more tags.
+
+### When to consider alternatives
+
+- When a badge is sufficient in place of a filter tag
+
+### Usage
+
+-
+
+### Accessibility
```

Comment:
Do we have documentation for these sections?

Reply:
@line47 So I left them blank for now because I was going to ask - do I just write this up or do we get official content from elsewhere?

Reply:
It might be good to work on some guidance for these and then get feedback from folks after we've written something up

Reply:
@line47 I've added some more guidance. But admittedly, writing code is more my thing than text for humans to read :)